disco_aws_automation/disco_deploy.py

```diff
@@ -579,7 +579,10 @@ def test_ami(self, ami, dry_run, deployment_strategy=None):
         # the testing environment but do need to be tested. An example would be hostclasses that are
         # only expected to exist in the bakery_environment. This doesn't hold true for update, which
         # should just always deploy things unless explicitly told no.
-        deployable = pipeline_hostclass_dict and self.is_deployable(hostclass)
+        if force_deployable is not None:
```

Comment:
Can this be `force_deployable or (pipeline_hostclass_dict and self.is_deployable(hostclass))`